UnityProject/Assets/Scripts/Gateway/StationGateway.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Mirror;
+using UnityEngine;
+
+/// <summary>
+/// For the station Gate, only connects to a gate this script will pick
+/// </summary>
+public class StationGateway : NetworkBehaviour
+{
+	[SerializeField]
+	private SpriteRenderer SpriteBaseTop, SpriteBaseBottom, SpriteBaseCentre, SpriteBaseRightTop, SpriteBaseRightMiddle, SpriteBaseRightBottom, SpriteBaseLeftTop, SpriteBaseLeftMiddle, SpriteBaseLeftBottom;
+	//TODO animate centre
```

Comment:
Try to make Spriterenderer[], if it works you will be able to write SetOnline, SetPowerOff and SetOffline methods nicely.

Reply:
ah thx

Reply:
Redone.